ivy/functional/frontends/jax/lax/control_flow_operators.py

```diff
@@ -58,3 +58,15 @@ def while_loop(cond_fun, body_fun, init_val):
     while cond_fun(val):
         val = body_fun(val)
     return val
+
+
+@to_ivy_arrays_and_back
+def scan(f, init, xs, length=None):
```

Comment:
This should take the same args as the original function, in this case (f, init, xs, length=None, reverse=False, unroll=1)

https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.scan.html